docs/spec/distribution/triggers.md

```diff
@@ -0,0 +1,31 @@
+# Triggers
+
+## Create validator distribution
+
+ - triggered-by: `stake.TxCreateValidator`
```

Comment:
Every time or only when the new validator candidate is bonded?

Reply:
(and in the latter case - then also by another validator unbonding, for example?)

Reply:
Also, do we need to anything when a validator is *unbonded*?

Reply:
>Every time or only when the new validator candidate is bonded?

Good observation, this only happens when the validator enters the bonded validator set. updated.

> Also, do we need to anything when a validator is unbonded?

**For other validators:**
When a validator is unbonded - the power change is be recorded and thus all other's passive calculations will incorporate the changes passively as expected. So nothing additional or special is required for an unbond for them. 

**For the affected validator:**
With regards to everyone affected by the unbond - we will need to incorporate a new mechanism to passively account for this - Just opened up an issue to think about this a bit more https://github.com/cosmos/cosmos-sdk/issues/1905.
I'm certain that the appropriate mechanism can be added onto what is spec'd here to account for this situation. THANK YOU FOR BRINGING THIS UP.